frontend/src/app/components/block-overview-graph/block-overview-graph.component.ts

```diff
@@ -656,7 +656,19 @@ export class BlockOverviewGraphComponent implements AfterViewInit, OnDestroy, On
 
   getFilterColorFunction(flags: bigint, gradient: 'fee' | 'age'): ((tx: TxView) => Color) {
     return (tx: TxView) => {
-      if ((this.filterMode === 'and' && (tx.bigintFlags & flags) === flags) || (this.filterMode === 'or' && (flags === 0n || (tx.bigintFlags & flags) > 0n))) {
+      let matches = false;
+      switch (this.filterMode) {
+        case 'and':
+          matches = (tx.bigintFlags & flags) === flags;
+          break;
+        case 'or':
+          matches = flags === 0n || (tx.bigintFlags & flags) > 0n;
+          break;
+        case 'nor':
+          matches = flags === 0n || (tx.bigintFlags & flags) === 0n;
```

Comment:
the first `flags === 0n` is kinda redundant (if true, then the following clause will also always be true)
```suggestion
          matches = (tx.bigintFlags & flags) === 0n;
```